app/main.py

```
@@ -19,12 +76,38 @@
+                    need_create_graph=False
+                )
+                active_datasets[row["id"]] = {
+                    "name": row["name"],
+                    "city_name": row["city"],
+                    "transport_type": row["transport_type"],
+                    "analysis_context": analysis_context,
+                    "user_id": row["user_id"],
+                    "guest_token": None,
+                }
+        finally:
+            break  # закрываем генератор после первого соединения
```

Ruff (0.14.8)

`break` inside `finally` blocks cause exceptions to be silenced

(B012)